e2e/playwright/support/ui/pages/boards_view.ts

```diff
@@ -21,6 +24,10 @@ export default class BoardsViewPage {
         this.topHead = page.locator('.top-head');
         this.editableTitle = this.topHead.getByPlaceholder('Untitled board');
         this.shareButton = page.getByRole('button', {name: '󰍁 Share'});
+        this.duplicateBoardButton = page.getByLabel('Duplicate board');
+        this.deleteBoardButton = page.getByLabel('Delete board');
+        this.deleteBoardConfirmationButton = page.locator('button.Button.filled.danger');
+        
```

Comment:
Same here, would be good to organize as delete modal component but good as is as well. Just pointing out for future consideration.
```suggestion
        this.deleteBoardConfirmationButton = page.locator('button.Button.filled.danger');
```